color-picker-project/src/components/navbar/Navbar.css

```diff
@@ -0,0 +1,120 @@
+.navbar {
+  background-color: white;
```

Comment:
For consistency we should define variables in the root selector in index.css for all of the colors we use in the project.  The advantage would be having all of the colors for the whole app in one place, which makes it much easier to make any changes.

We should also consider avoiding pure white (or black), and pick an off-white instead.